WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -92,10 +92,28 @@ class NotificationsViewController: UITableViewController, UIViewControllerRestor
     }()
 
     /// Notification Settings button
-    lazy var notificationSettingsButton: UIBarButtonItem = {
-        let button = UIBarButtonItem(image: .gridicon(.cog), style: .plain, target: self, action: #selector(showNotificationSettings))
-        button.accessibilityLabel = NSLocalizedString("Notification Settings", comment: "Link to Notification Settings section")
-        return button
+    lazy var notificationSettingsButton: UIButton = {
+        let settingsButton: UIButton = UIButton.init(type: .custom)
+        settingsButton.setImage(.gridicon(.cog), for: .normal)
+        settingsButton.addTarget(self, action: #selector(showNotificationSettings), for: .touchUpInside)
+        settingsButton.translatesAutoresizingMaskIntoConstraints = false
+        settingsButton.contentEdgeInsets = .zero
+        settingsButton.accessibilityLabel = NSLocalizedString("Notification Settings", comment: "Link to Notification Settings section")
+        return settingsButton
+    }()
+
+    /// Mark All As Read button
+    lazy var markAllAsReadButton: UIButton = {
+        let markButton: UIButton = UIButton.init(type: .custom)
+        markButton.setImage(.gridicon(.checkmark), for: .normal)
+        markButton.addTarget(self, action: #selector(markAllAsRead), for: .touchUpInside)
+        markButton.translatesAutoresizingMaskIntoConstraints = false
+        markButton.contentMode = .right
+        markButton.accessibilityLabel = NSLocalizedString(
+            "Mark All As Read",
+            comment: "Marks all notifications under the filter as read"
+        )
```

Comment:
Where can I find the localizations? I will also need the strings to display a TOAST/NoticeView upon completion of marking the notifications.

Reply:
Just following up here - as discussed in Slack, simply using NSLocalizedString is enough. The text will get translated later.